plugins/notifications-backend-module-slack/src/config.d.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface Config {
+  notifications: {
+    processors: {
+      slack: Array<{
+        /**
+         * Direct message workflow web request URL
+         * @visibility backend
+         */
+        directWebhookUrl?: string;
+        /**
+         * Channel message workflow web request URL
+         * @visibility backend
+         */
+        channelWebhookUrl?: string;
```

Comment:
These could be left as optional I think for if you want to configure only one of them (e.g. only group/channel support or only direct message support)

I'm fine with either -- optional or both required.

Reply:
Ok, optional is fine 👍